intranet/templates/eighth/admin/multiple_statistics.html

```diff
@@ -0,0 +1,115 @@
+{% extends "eighth/admin/eighth_admin_page_base.html" %}
+{% load static %}
+{% load pipeline %}
+
+{% block js %}
+    <script src="{% static 'vendor/sortable-0.8.0/js/sortable.min.js' %}"></script>
+    {{ block.super }}
+    <script>
+        $(function() {
+            Sortable.init();
+        });
+    </script>
+{% endblock %}
+
+{% block css %}
+    <link rel="stylesheet" href="{% static 'vendor/sortable-0.8.0/css/sortable-theme-minimal.css' %}">
+    {{ block.super }}
+{% endblock %}
+
+{% block admin_main %}
+    Filter student signups based on activities.
+    <form action="" method="post">
+        {% csrf_token %}
+        {{ form.as_table }}
+        <br>
+        <table>
+            <tr>
+                <th>Lower Limit:</th>
+                <td><input type="text" name="lower" value="{{ lower_absence_limit }}" placeholder="1"></td>
+
+                <td><input type="checkbox" name="freshmen" {% if include_freshmen %}checked{% endif %}> Freshmen</td>
```

Comment:
I think that it would be confusing to include the grade levels as a separate column to the right of the rest of the form; I think that putting the four on one row would be confusing compared to having the grade level checkboxes as a row.

So something like this:

```html
<tr>
    <th>Include:</th>
    <td>
        <input type="checkbox" name="freshmen"  id="freshmen" {% if include_freshmen %}checked{% endif %}>
        <label for="freshmen">Freshmen</label>
        <input type="checkbox" name="sophomores" id="sophomores" {% if include_sophomores %}checked{% endif %}>
        <label for="sophomores">Sophomores</label>
        ...
    </td>
<tr>
```

Reply:
The reason that I did it this way is it is consistent with the delinquent students view in eighth admin that 8PO mentioned in their email. I agree this probably would look better, but for the sake of consistency? Idk what do you think?

Reply:
> The reason that I did it this way is it is consistent with the delinquent students view in eighth admin that 8PO mentioned in their email. I agree this probably would look better, but for the sake of consistency? Idk what do you think?

To a new eighth period coordinator/admin I think it _could_ be confusing on the first few tries, but after someone knows how it is done, I don't think it's much of an issue.

Personally I'd change it here and on the "delinquent students view" as well to make it clearer.